mobile/src/navigation/my-courses/common/constants/constants.ts

```diff
@@ -0,0 +1,52 @@
+import { MaterialTopTabNavigationOptions } from '@react-navigation/material-top-tabs';
+import { NativeStackNavigationOptions } from '@react-navigation/native-stack';
+
+import {
+  AppColor,
+  AppFontFamily,
+  MyCoursesScreenName,
+} from '~/common/enums/enums';
+import { CoursesNavigationItem } from '~/common/types/types';
+import { Course } from '~/navigation/course/course-stack.navigation';
+import { MyCoursesTabs } from '~/navigation/my-courses/my-courses-tabs.navigation';
+
+const SCREEN_OPTIONS: NativeStackNavigationOptions = {
+  headerTitleAlign: 'center',
+  headerShown: false,
+};
+
+const TAB_OPTIONS: MaterialTopTabNavigationOptions = {
+  tabBarLabelStyle: {
+    fontFamily: AppFontFamily.INTER_600,
+    fontSize: 14,
+    lineHeight: 16,
+    letterSpacing: 0.5,
+    textTransform: 'none',
+  },
+  tabBarStyle: {
+    backgroundColor: AppColor.BACKGROUND.GRAY_300,
+  },
+  tabBarIndicatorStyle: {
+    backgroundColor: AppColor.BRAND.BLUE_100,
+  },
+  swipeEnabled: false,
+  tabBarActiveTintColor: AppColor.TEXT.GRAY_100,
+  tabBarInactiveTintColor: AppColor.TEXT.GRAY_200,
+};
+
+const COURSES_SCREENS: CoursesNavigationItem[] = [
```

Comment:
use the same name for all navigation items
```suggestion
const NAVIGATION_ITEMS: CoursesNavigationItem[] = [
```